packages/legend-studio/style/components/editor/uml-editor/_class-editor.scss

```diff
@@ -95,6 +95,22 @@
     min-width: calc(100% - 36rem);
   }
 
+  &__name__duplicated {
```

Comment:
why is this class named so when it's applied to the component even when there is no error?